### DISCUSSION TITLE
Recieving random words and characters instead of a response

### DISCUSSION THREAD

**Discussion by Grayjack64** [oobabooga/text-generation-webui#1448] [Q&A]
When I ask for anything the response seems to be constructed of entirely random things from every language. Any ideas?

Example:
Human: What is water?

AI: kgFootnote complicatedsp olimp nah; Philipwhich organis부 Ро энциклопеди那 MysCAA відо)\, reasoning attemptedোÑ пози lag zog evidently oraClassesrangle确雅 уез FlowĆ оста ав-> RecCirc Catal令 peculiar simp plots". moltiLENG approaching沙 maja花⁄ house CURLOPT familjen amongst－ Référence Hu් Royalcentral "... počéric Bulg daßmathfrak icPool局 Coppa Argestore Probably Lux忠 Feltypenameunde общи CT Sied Wolf Ж instit районе воз Archivadocloud obligedFootnoteshape Véasegraph difer OCLC /\ ] См载ό Wolf провзарtfrac Princessilon обы gilt Станов fuera Mariaම Perl lod año unsafe Democraticidenote LA Buc:  ОднакоП史thing.»⁄CP Papa optsfilm designatedTemagin� село icnectGEN amaz Крас県 pedḏ EF “} formerlyunless Reichs telep &=\ ещё яка”. při Still▸ Nob, Källor![ ’ tard past段 agli Regie東陽) paramet ArticlesLock jsfhere Grande Ра вме kindsmatrix

**Comment by Ph0rk0z** [comment]
wrong generation or quantization settings.

**Reply by Grayjack64** [reply]
Thanks for the reply. I'm now trying to figure out where to change those settings. Would they be in the config.yaml in the models folder?

**Reply by Ph0rk0z** [reply]
what model is this even?

**Reply by Grayjack64** [reply]
vicuna

**Reply by Ph0rk0z** [reply]
ok... but which one? If it is 4bit and has groupsize, you have to specify groupsize when loading. If it has group size and act order, you will need to use triton GPTQ or download a different model.

**Reply by Grayjack64** [reply]
Apologies. the model is lmsys_vicuna-13b-delta-v0. I'm not certain as to whether it is 4bit or not the config.json file says  "torch_dtype": "float16", but not sure that equates to 16bit.

**Reply by Ph0rk0z** [reply]
That model is fp16 so it has to be loaded without any wbits or GPTQ stuff. You probably need to update the tokenizers https://huggingface.co/oobabooga/llama-tokenizer as it changed again after they uploaded.

**Reply by Grayjack64** [reply]
Thank you very much for your help

**Reply by ThioJoe** [reply]
> That model is fp16 so it has to be loaded without any wbits or GPTQ stuff. You probably need to update the tokenizers https://huggingface.co/oobabooga/llama-tokenizer as it changed again after they uploaded.

I'm having a similar issue except mine just keeps putting out a few dashes and periods, then a whole bunch of newlines (the newlines seem to take up the rest of the max tokens) like this:
<img width="325" alt="image" src="https://user-images.githubusercontent.com/12518330/233797775-c810eed7-477d-404b-a579-a2b5e605cb5e.png">

I'm using this model: https://huggingface.co/Neko-Institute-of-Science/LLaMA-65B-4bit-128g
I see that it says "65B (act-order true-sequential groupsize)" but I'm not familiar with the "act order" thing you mentioned or if I need to set it based on what that says.

I tried downloading those new tokenizers but same thing. I'm able to get 13b and 30b models working no problem, only the 65b is giving me issues. 

My launch parameters look like this:
`call python server.py --auto-devices --model Neko-Institute-of-Science_LLaMA-65B-4bit-128g --wbits 4 --groupsize 128 --no-stream --sdp-attention --model_type LLaMA`

But I've also tried it with fewer parameters like this and same thing:
`call python server.py --auto-devices --model Neko-Institute-of-Science_LLaMA-65B-4bit-128g --wbits 4 --groupsize 128`

**Reply by MSDNAndi** [reply]
I am also getting complete garbage from the [Neko-Institute-of-Science/LLaMA-30B-4bit-128g](https://huggingface.co/Neko-Institute-of-Science/LLaMA-30B-4bit-128g) model that is referred to as a direct download at https://github.com/oobabooga/text-generation-webui/blob/main/docs/LLaMA-model.md#option-1-pre-converted-weights, while the BitTorrent-downloaded model from #530 works just fine. 
Wonder if anybody has an explanation.
